packages/adapters/cache/src/lib/caches/transactions.ts

```diff
@@ -104,21 +115,57 @@ export class TransactionsCache extends Cache {
     });
   }
 
-  public async storeTxData(txs: XTransfer[]): Promise<void> {
-    for (const tx of txs) {
+  public async storeTransfers(transfers: XTransfer[]): Promise<void> {
+    const nonceIncreasesByDomain: { [domain: string]: boolean } = {};
```

Comment:
can make the types a bit cleaner by doing: `Record<string, boolean>` but then its not labeled

Reply:
also maybe we can rename to `didNonceIncreaseByDomain` or something because `nonceIncreases...` makes it sound like you're storing the delta between nonces there

Reply:
ty